test/integration/cli/conftest.py

```diff
@@ -12,17 +12,89 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 from pathlib import Path
 from typing import Callable
 
 import pytest
+from id import (
+    AmbientCredentialError,
+    GitHubOidcPermissionCredentialError,
+    detect_credential,
+)
 
 from sigstore._cli import main
+from sigstore.oidc import _DEFAULT_AUDIENCE
 
 _ASSETS = (Path(__file__).parent.parent.parent / "assets/integration").resolve()
 assert _ASSETS.is_dir()
 
 
+def _has_oidc_id():
```

Comment:
This and below are duplicated from the unit-level conftest, right? I think this is fine for now, but we should probably think about deduping this at some point. Maybe some kind of `test/{common,unit,integration}` layout where `test.common` contains common helpers?